forum/investors/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
+from users.models import CustomUser
+from django.utils import timezone
 
-# Create your models here.
+
+class Investor(models.Model):
+    investor_id = models.AutoField(primary_key=True)
+    user_id = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
+    investor_logo = models.BinaryField(blank=True, null=True)
+    created_at = models.DateTimeField(default=timezone.now)
+    last_updated = models.DateTimeField(auto_now=True)
```

Comment:
Ensure the use of `BinaryField` for storing binary data like images is intentional. For images, you might want to use `ImageField` instead.

Reply:
Image filed requires images to be stored in local or external storage. Since we can limit the size of logo images, I think they can be stored in a binary field.
We can discuss it with the team.